src/lightkurve/interact.py

```diff
@@ -119,12 +119,15 @@ def prepare_tpf_datasource(tpf, aperture_mask):
         Bokeh object to be shown.
     """
     npix = tpf.flux[0, :, :].size
-    pixel_index_array = np.arange(0, npix, 1).reshape(tpf.flux[0].shape)
-    xx = tpf.column + np.arange(tpf.shape[2])
-    yy = tpf.row + np.arange(tpf.shape[1])
+    ncadences, ny, nx = tpf.shape
+    pixel_index_array = np.arange(0, npix, 1)
+    xx = tpf.column + np.arange(nx)
+    yy = tpf.row + np.arange(ny)
     xa, ya = np.meshgrid(xx, yy)
+    xa = xa.flatten()
+    ya = ya.flatten()
     tpf_source = ColumnDataSource(data=dict(xx=xa.astype(float), yy=ya.astype(float)))
-    tpf_source.selected.indices = pixel_index_array[aperture_mask].reshape(-1).tolist()
+    tpf_source.selected.indices = pixel_index_array[aperture_mask.reshape(-1)].tolist()
```

Comment:
in some lines, flatten is done by `.flatten()` while some others, it is done by `.reshape(-1)`. 
It'd be better if one form is used consistently.